airbyte-integrations/bases/base-normalization/setup.py

```diff
@@ -30,7 +30,7 @@
     author_email="contact@airbyte.io",
     url="https://github.com/airbytehq/airbyte",
     packages=setuptools.find_packages(),
-    install_requires=["airbyte-protocol", "dbt==0.18.2rc1", "pyyaml"],
+    install_requires=["airbyte-protocol", "pyyaml", "jinja2"],
```

Comment:
was `dbt` previously pulling in `jinja2` for us?

Reply:
yes exactly:

https://github.com/fishtown-analytics/dbt/blob/15e995f2f5ea18b333007c1fc2142e7dcf1c3abd/core/setup.py#L62